src/porepy/numerics/fv/biot.py

```diff
@@ -340,7 +297,17 @@ def discretize(self, sd: pp.Grid, sd_data: dict) -> None:
             "inverter", "numba"
         )
 
-        alpha: float = parameter_dictionary["biot_alpha"]
+        scalar_vector_mappings: dict = parameter_dictionary["scalar_vector_mappings"]
+        coupling_keywords: list[str] = list(scalar_vector_mappings.keys())
+
+        alpha: dict[str, pp.SecondOrderTensor] = {}
+
+        for key, alpha_input in scalar_vector_mappings.items():
+            # TODO: Revisit 'biot_coupling_coefficient
```

Comment:
Please search for TODOs before merging. Perhaps leave this as unresolved as a reminder?